packages/gasket-helper-intl/lib/index.js

```diff
@@ -149,6 +149,10 @@ function LocaleUtils(config) {
     const mappedLocale = localesMap && localesMap[locale] || locale;
     let fallbackLocale = mappedLocale;
 
+    if (locales && !locales[locale]) {
```

Comment:
Locales should be an array, not a map
```suggestion
    if (locales && !locales.includes(locale)) {
```

Add a test the ensure that it does NOT fallback to default if requested locale IS in locales list.